odoo/addons/test_http/utils.py

```diff
@@ -1,15 +1,51 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import geoip2.errors
+import geoip2.models
 from html.parser import HTMLParser
 from odoo.http import FilesystemSessionStore
 from odoo.tools._vendor.sessions import SessionStore
 
 
-class MemoryGeoipResolver:
-    def resolve(self, ip):
-        return {}
+ODOO_COM_IP = '178.33.40.43'
```

Comment:
since we have to mock the calls, could we *not* use a real routable IP here?
it's kind of confusing - does it need to be the real IP of odoo.com or not? what if the DNS changes? should someone make a PR? etc.

A test IP would seem appropriate, e.g. one from the documentation ranges 192.0.2.0/24, 198.51.100.0/24 or 203.0.113.0/24 [^1]

```suggestion
TEST_IP = '192.0.2.42'
```


[^1]: cfr [https://datatracker.ietf.org/doc/html/rfc5737](RFC5737)